app/controllers/accounts_controller.rb

```diff
@@ -108,4 +117,8 @@ def filtered_status_page(params)
       filtered_statuses.paginate_by_max_id(PAGE_SIZE, params[:max_id], params[:since_id]).to_a
     end
   end
+
+  def require_account!
```

Comment:
Right now (unless you've changed this) `AccountAccessConcern` is used for AP inboxes, and in the case of the shared inbox, `@account` is `nil`.

So I couldn't require the account there.

If you have a better idea for managing inboxes (maybe use a separate controller for the shared inbox that doesn't load the concern?) then feel free to move `require_account!` into there and I think it would be a good idea actually.